conda/models/records.py

```diff
@@ -411,6 +478,46 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.metadata = set()
 
+    @classmethod
+    def make_feature_record(cls, feature_name):
```

Comment:
Maybe we shouldn't really add this (as per the other comment above). Thoughts @jezdez?

Reply:
Nevermind, classic solver still uses it. We need to keep it.